### PR TITLE
Improving the approximate timing of reconstructed resonances.

### PR DESCRIPTION
**Pull request by jlconlin**

### CODE REVIEW

File: src/resonanceReconstruction/breitWigner/multilevel/Type/test/Type.test.cpp [hunk 1]
```diff
@@ -34,6 +34,7 @@ SCENARIO( "Integration test" ){
   SECTION( "Cobalt-58" ){
     auto Co58 = resonances("Co-58");
 
+    auto start = std::chrono::high_resolution_clock::now();
```
Comment (apmccartney) [inline src/resonanceReconstruction/breitWigner/multilevel/Type/test/Type.test.cpp]: you're conflating the parsing step and reconstruction step by beginning your timer here. If you're interested in comparing to `csmlbw`, you should move this to (what is now) line 52

File: src/resonanceReconstruction/breitWigner/multilevel/Type/test/Type.test.cpp [hunk 2]
```diff
@@ -50,11 +51,17 @@ SCENARIO( "Integration test" ){
...
+    auto microseconds =
+      std::chrono::duration_cast<std::chrono::microseconds>(finish-start);
+    njoy::Log::info( "Approximately {} microseconds passed while 'Apply'-ing",
+                    microseconds.count() );
   }
 
   SECTION( "Tulium-168" ){
     auto Tm168 = resonances("Tm-168");
 
+    auto start = std::chrono::high_resolution_clock::now();
```
Comment (apmccartney) [inline src/resonanceReconstruction/breitWigner/multilevel/Type/test/Type.test.cpp]: same

File: src/resonanceReconstruction/breitWigner/multilevel/Type/test/Type.test.cpp [hunk 3]
```diff
@@ -71,11 +78,17 @@ SCENARIO( "Integration test" ){
...
+    auto microseconds =
+      std::chrono::duration_cast<std::chrono::microseconds>(finish-start);
+    njoy::Log::info( "Approximately {} microseconds passed while 'Apply'-ing",
+                    microseconds.count() );
   }
 
   SECTION( "Neptunium-238" ){
     auto Np238 = resonances("Np-238");
 
+    auto start = std::chrono::high_resolution_clock::now();
```
Comment (apmccartney) [inline src/resonanceReconstruction/breitWigner/multilevel/Type/test/Type.test.cpp]: same

File: src/resonanceReconstruction/breitWigner/singleLevel/Type/test/Type.test.cpp [hunk 4]
```diff
@@ -21,12 +23,19 @@ const std::vector< double >& elastic();
 const std::vector< double >& capture();
 
 SCENARIO( "Integration test" ){
+  auto start = std::chrono::high_resolution_clock::now();
```
Comment (apmccartney) [inline src/resonanceReconstruction/breitWigner/singleLevel/Type/test/Type.test.cpp]: you're conflating the parsing step and reconstruction step by beginning your timer here. If you're interested in comparing to `csslbw`, you should move this to (what is now) line 31

File: src/resonanceReconstruction/reichMoore/Type/test/Type.test.cpp [hunk 5]
```diff
@@ -32,6 +32,7 @@ SCENARIO( "Integration test" ){
   SECTION( "Iron-56" ){
     const auto Fe56 = resonances("Fe-56");
 
+    auto start = std::chrono::high_resolution_clock::now();
```
Comment (apmccartney) [inline src/resonanceReconstruction/reichMoore/Type/test/Type.test.cpp]: you're conflating the parsing step and reconstruction step by beginning your timer here. If you're interested in comparing to `csrmat`, you should move this to (what is now) line 50

File: src/resonanceReconstruction/reichMoore/Type/test/Type.test.cpp [hunk 6]
```diff
@@ -47,17 +48,18 @@ SCENARIO( "Integration test" ){
...
       std::chrono::duration_cast<std::chrono::milliseconds>(finish-start);
-    std::cout << milliseconds.count() << " ms\n" << std::endl;
+    njoy::Log::info( "Approximately {} milliseconds passed while 'Apply'-ing",
+                    milliseconds.count() );
   }
 
   SECTION( "Uranium-235" ){
     const auto U235 = resonances("U-235");
 
+    auto start = std::chrono::high_resolution_clock::now();
```
Comment (apmccartney) [inline src/resonanceReconstruction/reichMoore/Type/test/Type.test.cpp]: same

File: src/resonanceReconstruction/reichMoore/Type/test/Type.test.cpp [hunk 7]
```diff
@@ -73,17 +75,18 @@ SCENARIO( "Integration test" ){
...
       std::chrono::duration_cast<std::chrono::milliseconds>(finish-start);
-    std::cout << milliseconds.count() << " ms\n" << std::endl;
+    njoy::Log::info( "Approximately {} milliseconds passed while 'Apply'-ing",
+                    milliseconds.count() );
   }
 
   SECTION( "Uranium-238" ){
     const auto U238 = resonances("U-238");
 
+    auto start = std::chrono::high_resolution_clock::now();
```
Comment (apmccartney) [inline src/resonanceReconstruction/reichMoore/Type/test/Type.test.cpp]: same

**Comment by jlconlin** [comment]
@apmccartney What is the status on this?

**Comment by jlconlin** [comment]
@apmccartney What do you say to this?

**Pull request closed without merging**